en/docs/learn/consume-api/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md

```diff
@@ -12,32 +12,84 @@ Make sure to take a backup of the `defaultTheme.js` before making any changes.
 
 Changes done in the `defaultTheme.js` file are reflected directly in the Developer Portal ( It's not required to restart the server or rebuild the source code).
 
+!!!Note
+    API Manager Devportal is themed using React Material Design. The theme configuration is an external JSON file resides outside the React codebase. While an administrator who has access to the file system can override the default theme configuration, a tenant admin can override both of them via the defaultTheme.json file. The parameters you can override via the theme are listed at the bottom. Additionally, the default theme params available with React Material Design library can be overridden via this file. Following link list down the available params  [https://material-ui.com/customization/default-theme/#default-theme](https://material-ui.com/customization/default-theme/#default-theme)
+
+
 ## Uploading via the Admin Portal (Tenants Only)
 
-**If you do not have access to the file system** , you can upload the theme via the Admin Portal as shown below:
+**If you do not have access to the file system** , you can upload the theme via the Admin Portal as shown below:
 
 1.  Download the sample theme here [sampleTheme.zip]({{base_path}}/assets/attachments/learn/sampleTheme.zip). 
 2.  The `sampleTheme.zip` file contains the following folder structure. 
 
      You can make the changes required to `defaultTheme.json` file and archive it back. The name of the archive does not matter, but the name of the JSON file (`defaultTheme.json`) does.
-     
+     Uses of resources in `login` folder will be discussed in ## Applying thems for tenant login pages section.
+
       ```
-      ├── css
-      │   └── custom.css
-      ├── defaultTheme.json
-      └── images
-          └── custom-logo.png
+      └──apim
+      │    └──css
+      │    │    └── custom.css
+      │    ├── defaultTheme.json
+      │    └── images
+      │          └── custom-logo.png
+      └──analytics
+      │     └── images
+      │          └── custom-logo.png
+      │          └── favicon.ico
+      └──login
+           └──css
+           │    └── custom.css
+           ├── loginTheme.json
+           └── images
+                 └── custom-logo.png
+                 └── favicon.ico
       ```
 
-3.  Sign in to the WSO2 Admin Portal ([https://server-host:9443/admin](https://server-host:9443/admin)) with your tenant username (format `<username>@<domain>.com kim@testorg.com`) and password.
+3.  Sign in to the WSO2 Admin Portal ([https://server-host:9443/admin](https://server-host:9443/admin)) with your tenant username (format `<username>@<domain>.com kim@testorg.com`) and password.
 
 4.  Expand the **Settings** menu, click **Upload Tenant Theme** and upload your ZIP file. 
 
-    ![Upload tenant theme]({{base_path}}/assets/img/learn/upload-tenant-theme.png)
+    ![Upload tenant theme]({{base_path}}/assets/img/Learn/upload-tenant-theme.png)
```

Comment:
@msm1992 , this folder path is incorrect as there is no folder named Learn/ anymore. Now all the folders are in lowercase.